test/clt-tests/sharding/syntax/queries-syntax.rec

```diff
@@ -8,9 +8,30 @@ mysql -h0 -P1306 -e "create table u(id bigint) shards=10 rf=1"; echo $?
 ––– output –––
 0
 ––– input –––
+mysql -h0 -P1306 -e "SHOW TABLES\G" | grep 'Index' | cut -d' ' -f 2 | sort
+––– output –––
+sharding_queue
+sharding_state
+sharding_table
+u
+u_s0
+u_s1
+u_s2
+u_s3
+u_s4
+u_s5
+u_s6
+u_s7
+u_s8
+u_s9
+––– input –––
+mysql -h0 -P1306 -e "create table t(id bigint) rf=1 shards=10"; echo $?
+––– output –––
+0
+––– input –––
 mysql -h0 -P1306 -e "SHOW TABLES\G"
```

Comment:
For cleaner output it's better to make it same with sort before and after

Reply:
It's not clear, explain or attach an example.

Reply:
I added the `| grep 'Index' | cut -d' ' -f 2 | sort` to both `"SHOW TABLES\G"` commands.